### PR TITLE
Add script to build and watch library and app

### PR DESCRIPTION
**Pull request by tuckerzp**
Using the concurrently package found at https://github.com/open-cli-tools/concurrently, this adds the ability to make
changes to components without needing to rerun to viewer to actually see the change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +47,8 @@
     "run-example": "cd example && npm start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "react-scripts": "react-scripts"
+    "react-scripts": "react-scripts",
+    "live-edit": "concurrently \"npm run start &\" \"cd example && npm run start\""
```
Comment (kylelaker) [inline package.json:51]: Okay so I don't want to bikeshed too much but with all the "build-blah-and-blah-blah" should this be something like `build-all-and-watch`

Comment (kylelaker) [inline package.json:51]: Do you need the `&` at the end of the first command here? The examples don't show it from the `concurrently` package.

File: package.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     "@mui/icons-material": "^5.8.0",
     "@mui/lab": "^5.0.0-alpha.82",
     "@mui/material": "^5.8.0",
+    "concurrently": "^7.2.1",
```
Comment (kylelaker) [inline package.json:23]: This needs to be a `devDependency`

File: package.json (hunk 1 above)
Comment (kylelaker) [reply package.json:51]: This is actually a question -- not necessarily definitive. Maybe casually poll some folks and see what the general feeling is?

Comment (tuckerzp) [reply package.json:51]: Sounds good! I was hoping for some input on the name.

Comment (tuckerzp) [reply package.json:51]: Oh I had copy pasted this command from my terminal, we probably do not need that.

Comment (Bronstrom) [reply package.json:51]: I'm trying to think of a better name without too much wording as well, but if we want this to follow the other name schemes maybe we name it something like `run-example-and-watch`?

**Comment by tuckerzp** [comment]
I apologize for `force-push` here. I needed to fix the merge conflicts as well as address review comments.

**Comment by kylelaker** [comment]
I don't _love_ pulling in a dependency just for a convenience command but in this case, `concurrently` seems popular enough and I think this command provides sufficient value to justify it.

**Pull request merged**